packages/extension-ui/src/components/Button.tsx

```diff
@@ -77,7 +88,7 @@ export default styled(Button)(
   width: ${tertiary ? 'max-content' : '100%'};
   height: ${tertiary ? 'unset' : '48px'};
   box-sizing: border-box;
-  border: none;
+  border: 1px solid transparent;
```

Comment:
Why transparent border, instead of e.g. a margin? 🧐

Reply:
Buttons were jumping on focus, because there we add 1px dashed border.
Margin could be overridden to get spacing, also I used border to fix border, which seems intuitive :grin: